vizro-core/docs/pages/user-guides/custom-css.md

```diff
@@ -0,0 +1,461 @@
+# Customizing Vizro dashboard CSS
+
+Vizro is opinionated about visual formatting, but it allows for some customization if you want to
```

Comment:
```suggestion
Vizro is opinionated about visual formatting, and some elements, such as the layout of the navigation and components, are fixed. You can customize some settings such as
```

Reply:
Suggested change to the intro to say first what is opinionated and fixed, and then what you can change. You can leave what you have if it's simpler though!